ros/setup

```diff
@@ -2,7 +2,14 @@
 
 if [ -z "$TUE_ROS_DISTRO" ]
 then
-    echo "[tue ros setup] TUE_ROS_DISTRO was not set"
+    echo "[tue ros install] TUE_ROS_DISTRO was not set"
+    echo "[tue ros install] Did you run the bootstrap-ros1.bash script?"
+    echo "[tue ros install] You can also set it manually like:"
+    echo ""
+    echo 'mkdir -p "${TUE_ENV_DIR}/.env/setup"'
+    echo 'echo "export TUE_ROS_DISTRO=noetic" > "${TUE_ENV_DIR}/.env/setup/user_setup.bash"'
+    echo 'source "${TUE_DIR}/setup.bash"'
+    echo ""
     return 1
```

Comment:
You shouldn't end up here, when the installer fails on this issue.

Reply:
And yet users sometimes do... It's hard to debug a partly-broken installation. This typically fixes it.